preprocessing_scripts/ert_rft.py

```diff
@@ -0,0 +1,196 @@
+import os
+import glob
+
+import pandas as pd
+from fmu.ensemble import ScratchEnsemble, EnsembleSet
+
+######################################################################
+# df_rft_obs: this function will make a dataframe of ERT rft obsdata
+# step 1
+######################################################################
+
+
+def get_rft_reportsteps(reportsteps):
+    """Combine rft data from multiple report steps"""
+    return pd.concat(
+        [
+            df_rft_obs(rs["folder"], rs["well_date"], rs["reportstep"])
+            for rs in reportsteps
+        ]
+    )
+
+
+def df_rft_obs(obsfolder, well_date_rft="well_date_rft.txt", reportstep=1):
+    """Extract observed RFT data from ERT inputfolder structure """
+    rft_wellname = []
+    rft_date = []
+    rft_meas = []
+    rft_meas_err = []
+    rft_east = []
+    rft_north = []
+    rft_md = []
+    rft_tvd = []
+    rft_zone = []
+
+    with open(os.path.join(obsfolder, well_date_rft)) as f:
```

Comment:
Recurring several places, but should we use pathlib instead? 
At least `os.path.join` returns it as a string and not a, which is enough to crash e.g. the Python debugger in vscode with a `FileNotFoundError`